addons/account/models/account.py

```diff
@@ -1122,6 +1129,8 @@ class AccountReconcileModel(models.Model):
         ('fixed', 'Fixed'),
         ('percentage', 'Percentage of balance')
         ], required=True, default='percentage')
+    tax_price_include = fields.Boolean(string='Is Tax Price Include', related='tax_id.price_include')
+    tax_include = fields.Boolean(string='Tax Include')
```

Comment:
use the same label as price_include field in account.tax => 'Tax Included in Price'